SPECS/quota/quota.spec

```diff
@@ -60,22 +60,6 @@ Source4:        rpc-rquotad.sysconfig
 Patch0:         quota-4.06-warnquota-configuration-tunes.patch
```

Comment:
Can we switch the `URL:` to https as well? `https://sourceforge.net/projects/linuxquota/`

Reply:
Good point!! URL updated to https.